tests/test_suite.py

```diff
@@ -0,0 +1,9 @@
+from sys import argv
+from test_functions import exit_tests, set_command_line_arguments
+
+set_command_line_arguments(argv[1:])
+
+# run the first time setup script
+import firsttimesetup
```

Comment:
using `import` to run code seems strange

Reply:
I thought that too- but it seems like the best way in Python.

Reply:
I thought you were planning on using something like pytest or other testing framework to organize these? That might make more sense

Reply:
Yeah, I have been looking into pytest and unittest. Not sure yet- I'll look into it more.